services/src/main/java/org/fao/geonet/api/records/MetadataUtils.java

```diff
@@ -759,9 +759,9 @@ public static void backupRecord(AbstractMetadata metadata, ServiceContext contex
                 BinaryFile.copy(is, os);
             }
         } catch (Exception e) {
-            Log.error(Geonet.GEONETWORK, "Backup record. Error: " + e.getMessage(), e);
+            throw new RuntimeException("Backup record. Error backing up record metadata.getUuid(): " + e.getMessage(), e);
```

Comment:
I guess this will cause the delete metadata service to fail if the backup can't be done?

As far as the user receives a proper message can be fine. I would add to the message to contact the system administrator to verify the problem if the problem persists.

Reply:
Yes, it will fail if the backup fails because it should not continue in this case otherwise, we will not have the backup that was expected.

I have updated the error message.

Reply:
Also, if there is some blocking exception, user can always use the withBackup API parameter (https://github.com/geonetwork/core-geonetwork/blob/main/services/src/main/java/org/fao/geonet/api/records/MetadataInsertDeleteApi.java#L199) if needed.